_maps/map_files/Delta/delta.dmm

```diff
@@ -14867,9 +14867,7 @@
 	},
 /area/quartermaster/office)
 "aTz" = (
-/obj/structure/table,
-/obj/item/clipboard,
-/obj/item/toy/figure/crew/qm,
+/obj/machinery/suit_storage_unit/qm/secure,
```

Comment:
![catexplode](https://github.com/ParadiseSS13/Paradise/assets/96800819/a92a1c90-05b0-4be3-9460-1da03b14671e)

mapping changes were kept to a minimum because of this

Reply:
In fairness it touches nothing in relation to the power grid and jut adds some console/machines while changing some door access.